dipy/reconst/shm.py

```diff
@@ -1012,7 +1053,8 @@ def __getitem__(self, index):
         return signal
 
 
-def sf_to_sh(sf, sphere, sh_order=4, basis_type=None, smooth=0.0):
+def sf_to_sh(sf, sphere, sh_order=4, basis_type=None, use_full_basis=False,
+             use_legacy_definition=False, smooth=0.0):
```

Comment:
I would simplify these variable names a bit:

```suggestion
def sf_to_sh(sf, sphere, sh_order=4, basis_type=None, full_basis=False,
             legacy=False, smooth=0.0):
```

Reply:
I agree, I'll remove them and update the documentation.

Reply:
Done, see 11beb24!